bayesian/utility/db_gateway.py

```diff
@@ -116,11 +116,20 @@ def get_batch_ca_data(cls, ecosystem: str, packages) -> dict:
         }
         return GraphAnalyses.post_gremlin(cls.ca_batch_query, bindings)
 
+    @classmethod
+    def get_batch_ca_vulnerability_data(cls, ecosystem: str, packages) -> dict:
```

Comment:
the name of the functions is same and is a bit confusing.

Reply:
ACK.